vscode/src/telemetry.ts

```diff
@@ -250,3 +254,19 @@ export function sendTelemetryEvent<E extends keyof EventTypes>(
     )}`,
   );
 }
+
+function getBrowserRelease(): string {
+  if (navigator.userAgentData) {
+    const browser =
+      navigator.userAgentData?.brands[
+        navigator.userAgentData.brands.length - 1
+      ];
+    return `${browser.brand}/${browser.version}`;
+  } else {
+    return "";
+  }
+}
+
+export function getUserAgent(): string {
+  return userAgentString || "";
```

Comment:
Shouldn't it fall back to 'navigator.userAgent' rather than an empty string?

Reply:
Not sure I follow. You set the User-Agent header to the value returned by this function, correct? (line 84 in networkRequests.ts). The "User-Agent" header on HTTP requests is by default the same value as navigator.userAgent, is my understanding:

Network trace:

<img width="1046" alt="image" src="https://github.com/microsoft/qsharp/assets/993909/1b0e775a-e598-401e-b8b3-fe67aff4693a">

'navigator.userAgent' value

<img width="925" alt="image" src="https://github.com/microsoft/qsharp/assets/993909/13d29932-546b-4357-bf05-a8fc3ab159bb">

So why would this be an invalid default if userAgentString doesn't have a value?

Or am I missing something?